src/PostgREST/ApiRequest/Preferences.hs

```diff
@@ -6,25 +6,30 @@
 --
 -- [1] https://datatracker.ietf.org/doc/html/rfc7240
 --
-{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE NamedFieldPuns  #-}
+{-# LANGUAGE RecordWildCards #-}
 module PostgREST.ApiRequest.Preferences
   ( Preferences(..)
   , PreferCount(..)
+  , PreferHandling(..)
   , PreferMissing(..)
   , PreferParameters(..)
   , PreferRepresentation(..)
   , PreferResolution(..)
   , PreferTransaction(..)
-  , PreferHandling(..)
   , fromHeaders
   , shouldCount
   , prefAppliedHeader
   ) where
 
 import qualified Data.ByteString.Char8     as BS
+import qualified Data.List                 as L
 import qualified Data.Map                  as Map
+import qualified Data.Text.Encoding        as T
```

Comment:
Just FYI, you don't need to import these since they are already included by Protolude. Calling the functions directly works OK.